blog-api-y-nguyen/Controllers/AuthorsController.cs

```diff
@@ -7,39 +7,43 @@
 using Microsoft.EntityFrameworkCore;
 using blog_api_y_nguyen.Models;
 using blog_api_y_nguyen.Repository;
+using blog_api_y_nguyen.Services;
+using AutoMapper;
 
 namespace blog_api_y_nguyen.Controllers
 {
     [Route("api/authors")]
     [ApiController]
     public class AuthorsController : ControllerBase
     {
-        private IAuthorRepository _authorRepository;
-        public AuthorsController(BlogContext context)
+        private readonly IAuthorService _authorService;
+        private readonly IMapper _autoMapper;
+        public AuthorsController(IMapper autoMapper, IAuthorService authorService)
```

Comment:
still not convinced why we need to use `autoMapper` here, will discuss with you later in monday